python/cuml/test/test_input_utils.py

```diff
@@ -66,6 +66,28 @@ def test_input_to_dev_array(dtype, input_type, num_rows, num_cols):
     del real_data
 
 
+@pytest.mark.parametrize('dtype', test_dtypes_acceptable)
+@pytest.mark.parametrize('input_type', test_input_types)
+@pytest.mark.parametrize('num_rows', test_num_rows)
+@pytest.mark.parametrize('num_cols', test_num_cols)
+def test_input_to_host_array(dtype, input_type, num_rows, num_cols):
+    input_data, real_data = get_input(input_type, num_rows, num_cols, dtype)
+
+    if input_data is None:
+        pytest.skip('cupy not installed')
```

Comment:
Hmmm... this seems like it would throw away errors that return none by accident. How about only skipping if cupy is really not installed? Don't we have a utility to check that?

Reply:
yeah, this test predates that utility, will update it